tasks/eth/021-holocene-protocol-versions/SignFromJson.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {ProtocolVersionsBump} from "script/verification/ProtocolVersionsBump.s.sol";
+import {SuperchainRegistry} from "script/verification/Verification.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {Proxy} from "@eth-optimism-bedrock/src/universal/Proxy.sol";
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+import {ProtocolVersions, ProtocolVersion} from "@eth-optimism-bedrock/src/L1/ProtocolVersions.sol";
+import {Types} from "@eth-optimism-bedrock/scripts/Types.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+
+contract SignFromJson is OriginalSignFromJson, ProtocolVersionsBump {
```

Comment:
We should make sure we understand why this script doesn't revert without overriding `getCodeExceptions()`. It [should revert](https://github.com/ethereum-optimism/superchain-ops/blob/3f63f70d868a3d02dff263e632c8cd55bf8255ac/src/JsonTxBuilderBase.sol#L135) without an override.

Reply:
Added a comment [5bfac3c](https://github.com/ethereum-optimism/superchain-ops/pull/428/commits/5bfac3cf807e4048a6630e06fd6207b7cbe6a838)

Reply:
Thanks nice work make more sense!